src/sass/_operation_list.scss

```diff
@@ -9,6 +9,10 @@
     }
   }
 
+  .date-pair {
+    text-wrap: auto;
```

Comment:
Isn't that the default? You need to override it?
Oh, you are overriding Vanilla's `pretty`? Maybe it would be worth mentioning in code comment for future context.